src/main/java/io/gravitee/policy/generatejwt/GenerateJwtPolicy.java

```diff
@@ -106,6 +107,10 @@ public void onRequest(Request request, Response response, ExecutionContext execu
                 signer = getSigner(hash);
 
                 JWSHeader.Builder builder = new JWSHeader.Builder(JWSAlgorithm.RS256).keyID(configuration.getKid());
+                if (configuration.getX5t() != null) {
+                    builder.x509CertThumbprint(new Base64URL(configuration.getX5t()));
```

Comment:
Thank you for this PR.

According to https://www.rfc-editor.org/rfc/rfc7515#page-12 the x5t is the fingerprint of the certificate used to sign the JWT. In that case, I thing this value should be computed based on the certificate provided into the configuration. 

So I think instead of a String, we should have a boolean to know if we have to compute this thumprint. (Or maybe an enum as we can have x5t using SHA-1 or x5t#256 using SHA-256)

what do you think ?

Reply:
yes, all valid points but unfortunately I'm not able to offer more since this PR is based on code-suggestion from a subcontractor and I don't have a development environment nor the knowledge to code this in Gravitee.

Reply:
Ok, in that case I will close this PR.
I opened this issue to implement this feature https://github.com/gravitee-io/issues/issues/8835.